src/errors.rs

```diff
@@ -4,39 +4,44 @@ use reqwest::{Error as ReqwestError, StatusCode};
 use thiserror::Error;
 
 #[derive(Error, Debug)]
-pub enum PushRelayError {
-    #[error("APNs error: {0}")]
-    Apns(#[from] A2Error),
+pub enum InitError {
+    #[error("APNs init error: {0}")]
+    Apns(#[source] A2Error),
```

Comment:
Regarding the change from `#[from]` to `#[source]` (here and also further below), do you explicitly want to avoid the `From<A2Error>` conversion?

From thiserror docs:

> The #[from] attribute always implies that the same field is #[source], so you don’t ever need to specify both attributes.

Reply:
Yes, I did want to avoid this because with `#[from]` it is quite hard to find any usages of the variant

Reply:
Fair 🙂